### ISSUE TITLE
error TS1086: An accessor cannot be declared in an ambient context.

### ISSUE THREAD

**Issue by deepender87** [zefoy/ngx-swiper-wrapper#260]
Hi I am facing this issue after doing npm install. Not sure why this is happening.
ERROR in ../node_modules/ngx-swiper-wrapper/lib/swiper.component.d.ts:21:9 - error TS1086: An accessor cannot be declared in an ambient context.

**Comment by deepender87** [comment]
`ERROR in ./node_modules/ngx-swiper-wrapper/fesm5/ngx-swiper-wrapper.js
Module build failed (from ./node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js):
TypeError: Cannot read property 'kind' of undefined
    at isAngularDecoratorMetadataExpression`

**Comment by deepender87** [comment]
Above error i got on building the application as prod. It is failing to build the prod version

**Comment by john20xdoe** [comment]
replicated with Angular 6 project using ng build prod

**Comment by drawcard** [comment]
Reporting this problem with Angular 8.3.23 

```bash
ERROR in ../node_modules/ngx-swiper-wrapper/lib/swiper.directive.d.ts:12:9 - error TS1086: An accessor cannot be declared in an ambient context.

12     set index(index: number);
           ~~~~~
../node_modules/ngx-swiper-wrapper/lib/swiper.component.d.ts:21:9 - error TS1086: An accessor cannot be declared in an ambient context.

21     get isAtLast(): boolean;
           ~~~~~~~~
../node_modules/ngx-swiper-wrapper/lib/swiper.component.d.ts:22:9 - error TS1086: An accessor cannot be declared in an ambient context.

22     get isAtFirst(): boolean;
           ~
```

@john20xdoe workaround fixed it for me

**Comment by agims7** [comment]
I have the same problem with angular in version 8,2,14

**Comment by alejandrombc** [comment]
I downgrade to version 8.0.2 and it worked (my angular version is on 8 too)

**Comment by ashish051321** [comment]
Yes, it worked for me as well, after downgrading the ngx-swiper-wrapper to 8.2.0.
I was using angular version 8.

**Comment by jadav1982** [comment]
I am getting below error. I am using angular version 9.

 node_modules/ngx-swiper-wrapper/lib/swiper.component.d.ts(21,9): error TS1086: An accessor cannot be declared in an ambient context.
    node_modules/ngx-swiper-wrapper/lib/swiper.component.d.ts(22,9): error TS1086: An accessor cannot be declared in an ambient context.
    node_modules/ngx-swiper-wrapper/lib/swiper.directive.d.ts(12,9): error TS1086: An accessor cannot be declared in an ambient context.
